### PR TITLE
ICML generation for LBF

### PR DESCRIPTION
**Pull request by mattleff**
Outstanding issues:

- [x] First paragraph custom style
- [x] Quotes, poetry custom style

### CODE REVIEW

File: .tooling/latex/.gitignore
Comment (bountonw) [inline .tooling/latex/.gitignore:1]: What does this change mean?

**Review by bountonw — APPROVED** [review]
Approving since I need to edit LBF22 this morning. However there is a lot I don't understand. Will we be needing to revert the poetry in PP to `>` ? Will this allow for nested lines?

**Comment by bountonw** [comment]
@mattleff Can you write an explanation of what this PR does when you merge so that we have it in the logs?

**Comment by mattleff** [comment]
> Approving since I need to edit LBF22 this morning. However there is a lot I don't understand. Will we be needing to revert the poetry in PP to `>` ? Will this allow for nested lines?

@bountonw I need to check with Kenton about the nested lines. Markdown supports nested block quotes (ie, `> > Whatever...`), but I don't know if that will translate well into InDesign or not. I've added a comment on #174 so I don't forget.

**Pull request merged**